src/styles/settings/_colors.scss

```diff
@@ -222,13 +222,13 @@ $YELLOW_800: #f9a825;
 // googleguidance@ and they said we're allowed to make reasonable tweaks to
 // the palette.
 $WEB_PRIMARY_COLOR: #3740ff;
-$WEB_SECONDARY_COLOR: #3fc4ff;
+// $WEB_SECONDARY_COLOR: #3fc4ff; // Not contrast conformant on white
```

Comment:
I think we should delete this if it's not used anywhere else.

Reply:
OK, deleted. Wasn't sure if you wanted to leave it as a placeholder for a replacement secondary color down the road.